pstore/kafka/backoff_test.go

```diff
@@ -0,0 +1,32 @@
+package kafka
+
+// TODO: If optiopay/kafka accepts pull request, remove this file.
```

Comment:
Thank you so much for reaching out to us about your forked repository, dropbox/kafka. We appreciate your willingness to share information. While I am sure that you have fixed many important issues that we have not yet run into, it seems that your forked, dropbox/kafka, repository has similar performance writing to our KAFKA cluster as the original optiopay/kafka.

Our problem is unique in that one of our 5 kafka nodes is down permanently. What I see with the round robin distributing producer is that we quickly do 4 writes. Then the writing hangs for like 5 minutes as it tries to write to the failed node. Then we do another 4 writes followed by another 5 minute hang as we try to write to the same failed node again. After 15 minutes, we do 12 writes. 

While I admit that this PR, doesn't fix the wide variety of issues that your forked repo does, this PR does fix our most pressing issue which is continually blocking on the same failed node in our cluster.

It may make sense to have this PR be a part of your forked repo. But I have to admit that what we are running into with the 1 down node may not be such a common use case.